client/components/Facilitator/Components/Cohorts/CohortDataTable.jsx

```diff
@@ -424,73 +386,194 @@ export class CohortDataTable extends React.Component {
+                            onClickNext={() => {}}
+                            onResponseChange={() => {}}
+                        />
+                    </Grid.Column>
+                    <Grid.Column className="cohortdatatable__scroll">
+                        <Table celled striped selectable role="grid">
+                            <Table.Header>
+                                <Table.Row>
+                                    <Table.HeaderCell scope="col" colSpan={2}>
+                                        {header || `Prompt: "${prompt}"`}
+                                    </Table.HeaderCell>
+                                </Table.Row>
+                            </Table.Header>
+                            <Table.Body>
+                                {rows.map((row, rowIndex) => {
+                                    let {
+                                        0: left = '',
+                                        [index + 1]: response = {}
+                                    } = row;
+                                    const isAudioContent = isAudioFile(
+                                        response.value
+                                    );
+                                    const { content = '' } = response;
+                                    const display = isAudioContent ? (
+                                        <React.Fragment>
+                                            {content ? (
+                                                content
+                                            ) : (
+                                                <audio
+                                                    src={`/api/media/${response.value}`}
+                                                    controls="controls"
+                                                />
+                                            )}
+                                            <Icon name="microphone" />
+                                        </React.Fragment>
+                                    ) : (
+                                        content
+                                    );
+
+                                    return (
+                                        <Table.Row
+                                            key={`modal-${slide.id}-${rowIndex}`}
+                                        >
+                                            <Table.HeaderCell verticalAlign="top">
+                                                {left}
+                                            </Table.HeaderCell>
+                                            <Table.Cell>
+                                                <p>{display}</p>
+
+                                                {display && (
+                                                    <Moment
+                                                        duration={
+                                                            response.created_at
```

Comment:
So Moment won't throw an error if `response.create_at` or `response.ended_at` don't exist?

Reply:
Interesting. I didn't consider that any data would appear in this table without those values, since created_at & ended_at are set when the user hits submit, which is also when the record gets made 🤷‍♂